frontend/views/containers/proposals/ProposalItem.vue

```diff
@@ -153,26 +198,30 @@ export default ({
     },
     statusDescription () {
       const votes = Object.values(this.proposal.votes)
+      const format = { year: 'numeric', month: 'long', day: 'numeric' }
       const yay = votes.filter(v => v === VOTE_FOR).length
       const nay = votes.filter(v => v === VOTE_AGAINST).length
       const total = yay + nay
       switch (this.proposal.status) {
         case STATUS_OPEN: {
           const excludeVotes = this.proposalType === PROPOSAL_REMOVE_MEMBER ? 1 : 0
-
-          return L('{count} out of {total} members voted.', {
+          const date = humanDate(this.proposal.data.expires_date_ms, format)
+          return L('{count} out of {total} members voted. Expires on {date}.', {
             count: votes.length,
-            total: this.groupMembersCount - excludeVotes
+            total: this.groupMembersCount - excludeVotes,
+            date
           })
         }
         case STATUS_FAILED: {
-          return L('Proposal refused with {nay} against out of {total} total votes.', { nay, total })
+          const date = humanDate(this.proposal.meta.dateClosed, format)
+          return L('Proposal rejected on {date} with {nay} against out of {total} total votes.', { nay, total, date })
```

Comment:
You're setting `dateClosed` on `proposal` up above in `group.js`, but are accessing it through `proposal.meta.dateClosed`. So this is happening:

<img width="614" alt="Screen Shot 2022-07-20 at 4 48 46 PM" src="https://user-images.githubusercontent.com/138706/179927656-1a412925-d4ec-441c-89ff-de82bbdd9398.png">

The lines above and below should read instead: `this.proposal.dateClosed` I think. Please verify.